src/adapters/node.ts

```diff
@@ -91,9 +101,10 @@ class NodeServer implements Server {
     if (!addr) {
       return;
     }
+    // Use https:// in URL if HTTPS is enabled
```

Comment:
i think your code is already readable in L107 :)

Reply:
removed